src/sources/tripadvisor/categories.rs

```diff
@@ -0,0 +1,12 @@
+/// Build a corresponding OSM class / subclass for a given pair of
+/// category / sub_category from TripAdvisor.
+pub fn get_class_subclass(
```

Comment:
I don't find any usage of the function, is it correct ?

Reply:
That's correct, I must have forgotten it after I rebased on your branch, or a refactoring of my own :c

Reply:
The file isn't even declared as a module so that definitly looks like a rebase artifact :innocent: